Modules/SubModule_Import/xASL_imp_NII2BIDS_Subject.m

```diff
@@ -1,13 +1,13 @@
-function xASL_imp_NII2BIDS_Subject(imPar, bidsPar, studyPar, nameSubject)
+function xASL_imp_NII2BIDS_Subject(imPar, bidsPar, studyPar, nameSubjectSession)
 %xASL_imp_NII2BIDS_Subject Run NII to ASL-BIDS for one individual subject.
 %
 % FORMAT: xASL_imp_NII2BIDS_Subject(imPar, bidsPar, studyPar, nameSubject)
 % 
 % INPUT:
-%   imPar           - JSON file with structure with import parameter (REQUIRED, STRUCT)
-%   bidsPar         - Output of xASL_imp_Config (REQUIRED, STRUCT)
-%   studyPar        - JSON file with the BIDS parameters relevant for the whole study (REQUIRED, STRUCT)
-%   nameSubject     - name of the subject (REQUIRED, LIST)
+%   imPar               - JSON file with structure with import parameter (REQUIRED, STRUCT)
+%   bidsPar             - Output of xASL_imp_Config (REQUIRED, STRUCT)
+%   studyPar            - JSON file with the BIDS parameters relevant for the whole study (REQUIRED, STRUCT)
+%   nameSubjectSession  - name of the subject (REQUIRED, LIST)
```

Comment:
`LIST` should be a `cell structure` right?

Reply:
I think so, fixed it 088eadb 👍